packages/iota/test/validation/taggedPayload.spec.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 IOTA Stiftung
```

Comment:
The filename should be `taggedDataPayload.spec.ts` as that is the name of what we are testing.